app/src/main/java/org/gdg/frisbee/android/api/model/FirebaseDynamicLinksResponse.java

```diff
@@ -0,0 +1,31 @@
+package org.gdg.frisbee.android.api.model;
+
+/**
+ * Created by unstablebrainiac on 26/5/17.
+ */
+
+public class FirebaseDynamicLinksResponse {
+    private String shortLink;
```

Comment:
Please mark the fields `final`. With this you need to remove the `setter` methods.